services/project-grafana-loki/metadata.yaml

```diff
@@ -5,6 +5,10 @@ category:
 type: platform
 scope:
   - project
+dependencies:
+  - project-logging
+  - grafana-loki
+  - logging-operator
```

Comment:
we can also add `rook-ceph-cluster` , but we can choose not to if we want to exclude transitive dependencies.

Reply:
I think we're sticking to keeping the dependencies defined here as "hard" dependencies where in the backend we actually prevent an app from being deployed altogether if the deps are not deployed (e.g. whenever we define `dependsOn`). Because for the other "soft" dependencies, they may only fail to deploy successfully on the default config. If we need both, I would vote to have them in separate fields